### DISCUSSION TITLE
Loading Seed Data that Differs by Env

### DISCUSSION THREAD

**Discussion by jduckwo** [theory/sqitch#743] [General]
We are looking for some help identifying strategies for using Sqitch to load seed data into a PostgreSQL database that can be different by environment. Does anyone here have any best practices they can recommend?

_Note: There will be completely different datasets that we need to load depending on the given environment._

**Comment by theory** [comment]
I tend not to use Sqitch for data loading, only schema loading, with a few minor exceptions. You could, however, use some combination of `\i` includes and an `:env_name` variable to point to the right file to load for a specific environment.

**Reply by jduckwo** [reply]
@theory Thanks for the help!!
